WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasPendingUpdateButton.lua

```diff
@@ -70,7 +70,25 @@ local methods = {
       WeakAuras.Import(self.companionData.encoded, nil, nil, linkedAuras)
     end
 
+    function self.callbacks.OnClickNormal()
+      self.menu = {}
+      for auraId in pairs(self.linkedAuras) do
+        if not self.linkedChildren[auraId] then
+          tinsert(
+            self.menu,
+            {
+              text = auraId,
+              notCheckable = true,
+              func = function() WeakAuras.PickDisplay(auraId, "information") end
+            }
+          )
+        end
+      end
+      LibDD:EasyMenu(self.menu, WeakAuras_DropDownMenu, self.frame, 0, 0, "MENU")
```

Comment:
That menu looked a bit odd, did you consider adding a title to it, and/or making it jump to the aura if there's only one aura linked?